chapters/core/guides/index.rst

```diff
@@ -4,43 +4,52 @@ Jina developer guides cover specific features and use cases.
 
 Jina Core is the open-source framework of Jina AI. It is a new, state-of-the-art design pattern allowing you to create end-to-end search solutions. It leverages the latest AI and deep learning models out-of-the box and lets you deploy them anywhere you want.
 
-Brand new users to Jina Core should first begin with the :doc:`../introduction/index` section which covers more fundamental concepts.
+New users to Jina Core should first begin with the :doc:`../introduction/index` section which covers more fundamental concepts.
 
 Further information on any specific method can be obtained in the :doc:`../api_references/index`.
 
 
 .. toctree::
+   :caption: Understanding Jina
    :titlesonly:
-   :glob:
+
 
    ../setup/index
    ../../my_first_jina_app
    ../../cross_multi_modality
    ../../crud
-   ../../logging
-   ../../prevent_duplicate_indexing
-   ../../flow/index
    ../../flow/pattern
    ../../input_output
    ../../traversal
    ../../extend/*
-   ../../remote/index
-   ../../parallel
    ../../primitive_data_type
    ../../project_guide
    ../../traversal/index
-   ../../simple_exec
    ../../cli/exit
    ../../envs
-   ../../project-guide/project-guide
-   ../../optimization
-   ../../batching
-   ../../request_size
-   ../../evaluation
    ../../ranker
-   ../../incremental_indexing
-   ../../stress/index
    ../../mime-type/index
    ../../executors
    ../../peas-and-pods
 
+.. toctree::
+   :caption: Expanding Jina
```

Comment:
Good point! Instead of **optimising** I would go with **improving**. Optimizing sounds very fancy and very specific, and some of these guides are very generic